debian/messaging-app-apparmor.manifest

```diff
@@ -27,8 +27,10 @@
         "/usr/share/applications/",
         "/custom/xdg/data/dconf/",
         "/usr/share/*/assets/",
+        "@{PROC}/[0-9]*/task/[0-9]*/comm",
```

Comment:
I don't think exposing the command of every process' threads to every other process is a good idea. This would allow any app to learn all the other running apps and services on a system. If this is really necessary, try to find a way to limit this to the `self` PID instead. To find out if it is necessary, browse the Qt source code to learn what is actually trying to use `/proc/[pid]/task/[tid]/comm`.

Reply:
Is this piece anymore requiring work from my end since the Qt revision you mention in the other conversation, or is it unrelated and I should drop that piece and work this out?

Reply:
I suspect that the use of the command file in proc is for a different feature in the Qt renderer. If all applications had permission to the shader cache directory but did not have access to the command, did the shader cache still work?

If you still want to figure out why Qt is using the command file and what that breaks, it'd be cool.